zerver/tests/test_notification_data.py

```diff
@@ -212,6 +212,24 @@ def test_is_notifiable(self) -> None:
         user_data = self.create_user_notifications_data_object(user_id=user_id, pm_push_notify=True)
         self.assertTrue(user_data.is_notifiable(acting_user_id=acting_user_id, idle=True))
 
+    def test_bot_user_notifiability(self) -> None:
+        # Non-bot user (`user_id=9`) should get notified, while bot user (`user_id=10`) shouldn't
+        for user_id, notifiable in [(9, True), (10, False)]:
```

Comment:
These IDs seem rather fragile.  Can we iterate over email addresses, and look up their IDs in the loop?